cis194/week1/daniel-deng/src/CIS194/Homework01/Exercise05.hs

```diff
@@ -0,0 +1,23 @@
+module CIS194.Homework01.Exercise05 (hanoi) where
+
+type Disks = Integer
+type Peg   = String
+type Move  = (Peg, Peg)
+
+{-
+  To move n discs (stacked in increasing size) from peg a to peg b
+  using peg c as temporary storage,
+  1. move n − 1 discs from a to c using b as temporary storage
+  2. move the top disc from a to b
+  3. move n − 1 discs from c to b using a as temporary storage.
+-}
+hanoi :: Disks -> Peg -> Peg -> Peg -> [Move]
+hanoi 0 _ _ _                  = []
+hanoi 1 start destination _    = [(start, destination)]
+hanoi n start destination temp
+  | 0 > n     = []
+  | otherwise =
+      hanoi n' start temp destination ++
```

Comment:
In Haskell the convention is to align symbols, so here we would normally align ++ and equal sign above so that ++s are at the beginning of each line

Reply:
Done. Thanks for pointing it out!